gradle/libs.versions.toml

```diff
@@ -15,9 +15,9 @@
 # specific language governing permissions and limitations
 # under the License.
 [versions]
-fabric8 = "7.3.0"
+fabric8 = "7.3.1"
```

Comment:
Thank you for making a PR, but please revert this, @csviri . You can do this upgrade independently.

Reply:
Ur, wait, your PR looks strange. We already have this.

- #229